tools/ports/sqlite3.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+import os
+import shutil
+import logging
+
+TAG = '3380500'
```

Comment:
Can we pick a named release?

Reply:
3380500 is for release 3.38.5

Reply:
I have modified the code to use a version tuple.